### ISSUE TITLE
Compile it on mac

### ISSUE THREAD

**Issue by Dazzid** [sebastiandev/zipper#13]
Hi Sebastiandev
I am searching for a c++ zip generator and zipper seems to be the solution. 
It is posible to compile it with mac? 
Thanks for the help
David

**Comment by sebastiandev** [comment]
@Dazzid sorry for the delay, i think @fbergmann has been using this quite a lot and in many platforms. Were you able to get it running on mac?

We might need to merge @fbergmann pull request in order to make it fully compatible with mac, if thats the case then i'll merge it. But ill wait on his answer to be sure.

**Comment by fbergmann** [comment]
Hello, indeed the project with all the changes I made works fine on windows, linux and osx. However it no longer uses the nice C++ 11 features (which i had to drop in order to be able to run on older linux versions). If you like I can make a new pull request ... if you like ... 

again thanks a lot for the great project it helped a lot. 

Frank

**Comment by sebastiandev** [comment]
@Dazzid  @fbergmann I have just merged the last pull request, and updated docs as well.
@Dazzid please let me know if this works for you, and if it does then I'll close this issue.

Thanks!

**Comment by sebastiandev** [comment]
@Dazzid can I close this?

**Comment by Dazzid** [comment]
@sebastiandev @fbergmann Sorry for the delay! and thanx a lot for the help, it is working! 
Best
